Code/Yahoo.Yui.Compressor.Tests/TestHelpers/BuildEngine.cs

```diff
@@ -8,29 +9,34 @@ public static class BuildEngine
     {
         public static bool ContainsError(IBuildEngine engine, string error)
         {
-            return CheckForError(engine, error, true);
+            return CheckForError("Errors", engine, error, true);
         }
 
         public static bool DoesNotContainError(IBuildEngine engine, string error)
         {
-            return CheckForError(engine, error, false);
+            return CheckForError("Errors", engine, error, false);
         }
 
-        private static bool CheckForError(IBuildEngine engine, string error, bool exists)
+        public static bool ContainsMessage(IBuildEngine engine, string message)
+        {
+            return CheckForError("Messages", engine, message, true);
+        }
+
+        private static bool CheckForError(string messageType, IBuildEngine engine, string searchString, bool exists)
         {
             var buildEngine = engine as BuildEngineStub;
             if (buildEngine == null)
             {
                 Assert.Fail("Not a BuildEngineStub, cannot test with this");
             }
-            if (buildEngine.Errors != null && buildEngine.Errors.Count > 0)
+
+            var outputStrings = messageType == "Errors" ? buildEngine.Errors : buildEngine.Messages;
+
+            if (outputStrings.Count > 0)
             {
-                foreach (var anError in buildEngine.Errors)
+                if (outputStrings.Any(anError => anError.StartsWith(searchString)))

```

Comment:
This is no longer an error so anError is a poor name for the variable.  With the previous refactor to use messages instead of outputStrings, change this line to:
`if (messages.Any(message => message.StartsWith(searchString)))`